python/py-configparser/Portfile

```diff
@@ -24,18 +24,23 @@ master_sites        pypi:[string index ${python.rootname} 0]/${python.rootname}
 
 distname            ${python.rootname}-${version}
 
-checksums           rmd160  9b6a5b12490e72c47ce2b1d190a35687c2d2c7eb \
-                    sha256  c7d282687a5308319bf3d2e7706e575c635b0a470342641c93bea0ea3b5331df \
-                    size    72498
+checksums           rmd160  4ff3a2f5f7313e1df0459a7a75b114acbd4caf58 \
+                    sha256  85d5de102cfe6d14a5172676f09d19c465ce63d6019cf0a4ef13385fc535e828 \
+                    size    71248
 
 # 4.0.2 is the last py27-compatible version
 livecheck.type      none
 
 if {${name} ne ${subport}} {
     depends_build-append \
                         port:py${python.version}-setuptools \
-                        port:py${python.version}-setuptools_scm
-    depends_lib-append  port:py${python.version}-backports
+                        port:py${python.version}-setuptools_scm \
+                        port:py${python.version}-toml \
+                        port:py${python.version}-wheel
```

Comment:
this shouln't need `py-wheel` - correct? It's used in `pyproject.toml` but that's only for when you want to build `.whl` packages I think. Did you try without it?

Reply:
Removed